client/mysqlcheck.c

```diff
@@ -437,6 +437,9 @@ static int get_options(int *argc, char ***argv)
   int ho_error;
   DBUG_ENTER("get_options");
 
+  if (current_host == NULL)
+    current_host = getenv("MARIADB_HOST");
```

Comment:
Style fix (applies to all other code too),

`current_host= getenv...`

No space between var and `=`.